docs/contributing/Building, Debugging, and Testing on Unix.md

```diff
@@ -8,8 +8,18 @@ Particularly for developers who aren't experienced with .NET Core development on
 1. Run `./build.sh --restore`
 1. Run `./build.sh --build`
 
+## Working in Visual Studio Code
+1. Install [VS Code](https://code.visualstudio.com/Download)
+    - After you install VS Code, install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+    - Important tip: You can look up editor commands by name by hitting *Ctrl+Shift+P*, or by hitting *Ctrl+P* and typing a `>` character. This will help you get familiar with editor commands mentioned below. On a Mac, use *⌘* instead of *Ctrl*.
+2. Install a recent preview [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core). At time of writing, Roslyn uses .NET 5 preview 7. The exact version in use is recorded in our [global.json](https://github.com/dotnet/roslyn/blob/master/global.json) file.
```

Comment:
Hmm, worried this creates another place where we have to remember to update the SDK info when we move to new SDKs. Not sure how to fix this.

Reply:
Perhaps we should just make these 2 documents 1 document, with separate Windows and Unix sections in it. Then we can just have 1 section in that document with "Install the .NET SDK (windows link, mac link, linux link)"?

Reply:
We reference global.json as our source of truth here, as we do in our build scripts. I wrote this assuming the documentation will fall out of date with the truth. We can either refrain from referencing a specific version in this document, or we can write a script that fails the build if documentation references a different SDK version than is referenced in global.json :P

Reply:
> Perhaps we should just make these 2 documents 1 document

I think this might be reasonable to do in a future PR.